### PR TITLE
Remove a wrong call that disable lights on F3D renderer

### PR DESCRIPTION
**Pull request by mwestphal**
Fix https://github.com/f3d-app/f3d/issues/605

### CODE REVIEW

File: plugins/assimp/module/vtkF3DAssimpImporter.cxx [hunk 1]
```diff
@@ -88,8 +88,6 @@ class vtkF3DAssimpImporter::vtkInternals
 
       if (nbLights > 0)
       {
-        renderer->AutomaticLightCreationOff();
```
Comment (Meakk) [inline plugins/assimp/module/vtkF3DAssimpImporter.cxx:91]: I remembered doing that on purpose. The idea is to remove VTK lights when there is at least one light provided in the file.

Comment (mwestphal) [reply plugins/assimp/module/vtkF3DAssimpImporter.cxx:91]: not needed:

https://github.com/Kitware/VTK/blob/master/Rendering/OpenGL2/vtkOpenGLRenderer.cxx#L141

Comment (mwestphal) [reply plugins/assimp/module/vtkF3DAssimpImporter.cxx:91]: (and light test would fail if that were not true.)

Comment (mwestphal) [reply plugins/assimp/module/vtkF3DAssimpImporter.cxx:91]: (and the renderer parameters should not be changed by the importer anyway)

**Review by Meakk — APPROVED** [review]

**Pull request merged**